aws/lambda/contactFormSending.js

```diff
@@ -0,0 +1,37 @@
+const AWS = require('aws-sdk');
+
+const ses = new AWS.SES();
+
+const RECEIVER = 'events@brumtechtapas.co.uk';
```

Comment:
I mean we -could- put this in the .env file, right? Seems like it's the right place for it